templates/repo/pulse.tmpl

```diff
@@ -130,7 +134,7 @@
 {{end}}
 
 {{if gt .Activity.MergedPRCount 0}}
-	<h4 class="divider divider-text tw-normal-case" id="merged-pull-requests">
+	<h4 class="divider divider-text tw-normal-case tw-my-5" id="merged-pull-requests">
```

Comment:
two things in my mind:

1. why not add a general padding for something like `h4.divider` or `h4.divider-text`
2. `tw-normal-case` should be a no-op now since the fomantic styles have been removed?

Reply:
both done, there is no more `text-transform` in first-party css and only two more cases in fomantic css.